### ISSUE TITLE
Can not start com.apple.debugserver on iOS 14 device under Xcode 12.0 beta 4

### ISSUE THREAD

**Issue by albertdai** [ios-control/ios-deploy#469]
## MUST READ BEFORE YOU FILE (DELETE THIS SECTION BEFORE FILING)

Include the **command line arguments** you used for ios-deploy.

Don't forget to check out the [README](https://github.com/phonegap/ios-deploy/blob/master/README.md) before filing this issue

The same thread in the libimobiledevice side: https://github.com/libimobiledevice/libimobiledevice/issues/1004

# Expected behavior

Install the app successfully and launch the app on the device

# Actual behavior.

The command failed with "Error 0xe8000022: The service is invalid. AMDeviceSecureStartService(device, CFSTR("com.apple.debugserver"), NULL, &con)"

# Steps to reproduce the problem

1. Connect to an iOS 14 device and use Xcode 12.0 beta4
2. Run `ios-deploy -d -b {Bundle.app}` 

# System Specs

macOS 15.5
Xcode 12.0
Build version 12A8179i

Please run the commands below in your Terminal.app and include it in the issue. Check when done and include results below.

- [ ] 1. system_profiler SPSoftwareDataType
- [ ] 2. ios-deploy -V
- [ ] 3. xcodebuild -version
- [ ] 4. xcode-select --print-path
- [ ] 5. gcc --version
- [ ] 6. lldb --version
- [ ] **If using NodeJS/npm**
  - [ ] 7. npm -v
  - [ ] 8. node -v

**Comment by gabebear** [comment]
I haven't tested beta4 yet, this may be related to #468

When Xcode12 hits GM I will look into this and #468 further.

**Comment by albertdai** [comment]
The workaround/fix in libimobiledevice side could be replacing the service name `com.apple.debugserver.DVTSecureSocketProxy` and removing one line about SSL disable. See details in https://github.com/libimobiledevice/libimobiledevice/issues/1004.

But I could confirm that only replacing the service name in ios-deploy could not fix the issue. May need more change same as the line about SSL in libimobiledevice

**Comment by xster** [comment]
Seems like https://github.com/libimobiledevice/libimobiledevice/pull/1012/files was a working solution for libimobiledevice. @gabebear would you be able to map it to ios-deploy?

**Comment by ivanhernandez13** [comment]
I took an initial look into this and I don't think that fix will map over to ios-deploy because that fix looks like it is done at a layer below where ios-deploy operates i.e. configuring and enabling SSL is done at the MobileDevice framework level.

ios-deploy might be able to drop down to that level to create and configure the connection to the device with SSL by following what other third_party tools do but i think the MobileDevice framework might already do this. I spent some time this morning into looking MobileDevice to see if there's something we're missing that tells it to establish an SSL connection with the device. Will look further in the next couple of days. 

Been referencing this reimplementation of the framework: https://github.com/samdmarshall/SDMMobileDevice/blob/c3e1e97b1310c7a7a10f68281752760038b75e16/Framework/src/SDMMD_Connection.c#L194
